install-deps.sh

```diff
@@ -339,7 +339,10 @@ else
         case "$ID" in
             fedora)
                 $SUDO dnf install -y dnf-utils
-                $SUDO dnf install -y docker-ce docker-ce-cli containerd.io
+                dnf list installed | egrep "docker-ce|docker-ce-cli|containerd\.io|podman"
```

Comment:
please use the promtools shipped by fedora instead of using a container for it, since fedora comes with a more recent prometheus.